pom.xml

```diff
@@ -218,9 +218,6 @@
     <maven.versions.plugin.version>2.19.1</maven.versions.plugin.version>
     <maven.compiler.plugin.version>3.14.1</maven.compiler.plugin.version>
     <checkstyle.skipCompileInputResources>true</checkstyle.skipCompileInputResources>
-    <checkstyle.openrewrite.version>1.0.0-SNAPSHOT</checkstyle.openrewrite.version>
```

Comment:
Failure related to this update.
This project is not released to maven central, so it rely on pre install to local maven repository

Reply:
inline can not change its about the profile stuff. We having big issues not having a released version. I like to use it early but it might be too early as its imposing burden to handle rewrite.